client/src/components/common/Button/ContainedButton.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Button from '@material-ui/core/Button';
+import { withStyles } from '@material-ui/core/styles';
+
+const styles = {
+  root: {
+    appearance: 'none',
+    background: 'hsl(22, 85%, 53%)',
+    borderRadius: '0.3rem',
+    border: 0,
+    color: 'white',
+    height: 48,
+    padding: '0 2rem',
+    minWidth: '8rem',
+    '&:hover': {
+      backgroundColor: 'hsl(22, 85%, 45%)',
+    },
+  },
+};
+
+function ContainedButton(props) {
+  const { classes, label } = props;
+  return <Button className={classes.root}>{label}</Button>;
+}
+
+ContainedButton.propTypes = {
+  // eslint-disable-next-line react/forbid-prop-types
+  classes: PropTypes.object.isRequired,
```

Comment:
class Name should be string, so we can pass one class for our button as prop & we can set a general class for all our buttons 
``` className={`btnStyle ${className}`}  ```
```suggestion
   className: PropTypes.string.isRequired,
```